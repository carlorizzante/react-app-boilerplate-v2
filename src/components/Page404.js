import React from 'react';
import { Link } from 'react-router-dom';

const Page404 = (props) => (
  <div>
    <h1>404!!</h1>
    <p>Oops... no content on this page.</p>
    <p><Link to="/">Back to home</Link></p>
  </div>
);

export default Page404;
