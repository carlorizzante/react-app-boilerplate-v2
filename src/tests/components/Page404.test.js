import React from 'react';
import { shallow } from 'enzyme';
import Page404 from '../../components/Page404.js';

test('should render a 404 page', () => {
  const wrapper = shallow(<Page404/>);
  expect(wrapper).toMatchSnapshot();
});
