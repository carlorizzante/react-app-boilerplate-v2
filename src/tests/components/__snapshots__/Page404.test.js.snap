// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a 404 page 1`] = `
<div>
  <h1>
    404!!
  </h1>
  <p>
    Oops... no content on this page.
  </p>
  <p>
    <Link
      replace={false}
      to="/"
    >
      Back to home
    </Link>
  </p>
</div>
`;
